neon_audio/cli.py

```diff
@@ -70,33 +73,22 @@ def run(module, package, force_install):
     main()
     click.echo("Audio Client Shutdown")
 
-
-@neon_audio_cli.command(help="Install a TTS Plugin")
-@click.option("--module", "-m", default=None,
-              help="TTS Plugin to configure")
-@click.option("--package", "-p", default=None,
-              help="TTS package spec to install")
-@click.option("--force-install", "-f", default=False, is_flag=True,
-              help="Force pip installation of configured module")
-def install_plugin(module, package, force_install):
-    from neon_audio.utils import install_tts_plugin
```

Comment:
Please add a deprecation log so anyone using the old entrypoint gets a hint for how to update calls
```python
log_deprecation("`install-plugin` replaced by `install-dependencies`, "2.0.0")
```